### PR TITLE
Popup window: Dynamic JavaScript

### PR DESCRIPTION
**Pull request by SiveMdluli**
##PR
- Use javascript to dynamically create a popup window
- Onclick of an HTML button, trigger a modal for the respective card.
- Style to fit in both mobile and desktop

### CODE REVIEW

**Comment by NahnahAJ** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @SiveMdluli , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that has some linter checks failing. Killing ensure that your project is devoid of linter errors 💯 

56299674-3D1D-40F8-9CBA-273B9D6EC629

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -262,6 +260,187 @@ <h2 class="contact-me-text">Contact me</h2>
...
+
+                    <hr class="popup-hr">
+                    <div class="button">
+                        <button type="button" class="see-button seeLive">See live  <img src="icons/Icon.svg"/></button>
+                        <button type="button" class="see-button seeSource">See source <img src="icons/Vector.svg"/></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (NahnahAJ) [inline index.html:442]: - [ ] Just as i mentioned in the comment before, per the project requirements, the `modal popup` is to be populated with the data from the projects array after you are done generating the project section dynamically with data from the data array 💯 
- So kindly remove all of this content from the `index.html` file 👍

File: script.js [hunk 2]
```diff
@@ -1,35 +1,167 @@
...
-  sideNav.style.width = "0";
+  document.getElementById('sideNav').style.width = '0';
+  sideNav.style.width = '0';
   return close;
 }
 
-// CONTACT VALIDATION
+document.getElementById('sideNav').onclick(closeNav);
+
+const popup = document.getElementById('popupCard');
```
Comment (NahnahAJ) [inline script.js:19]: - [ ] In the mobile view, your design doesn't look completely like the Figma design. Kindly refer to the image below and make the necessary changes while referring to the Figma Design 💯 

- Your Design
F5570B8E-B390-4C8B-8C7E-6C53165FD929
- The Figma Design
0F7DB58B-62F9-404E-8964-B05B5749FE3D_4_5005_c

- [ ] In the desktop view, there are a few style issues that need to be fixed as well. I have pointed them out in the screenshot below 👍 
- Your Design
077006AC-9F18-4F9A-9454-F87802328778

- The Figma Design
A17286D4-6AEA-43FA-8F4C-2CC50A76030B_4_5005_c

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @SiveMdluli and @unorjikingsley , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
-Good use of GitHub flow ✔️ 
- No linter errors ✔️ 
- Professional README ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lookingfor00 — COMMENTED** [review]
Hi @SiveMdluli , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that was created last week, as this is a pair programming activity, you need to create a new PR with your work and your new partner's work.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**